src/types/factories/PolicyFactory.d.ts

```diff
@@ -10,10 +10,9 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+import ModelResolutionFactory from '../resolution-factory';
 
-/**
- *  THIS FILE IS AUTO-GENERATED - SEE CONTRIBUTOR DOCUMENTATION
- */
```

Comment:
This does raise an issue - these files are autogenerated, yes?  Can we add a comment to that extent?  I lost a day recently in another codebase editing a generated file and it is not fun to discover.

Reply:
Handlebars files are not autogenerated but they contain this banner so it gets into generated `.d.ts` and `.js` files. This template is deleted because there seem to be no need in having index file for model types.

It makes sense to have it for for JS files so all models can be [imported](https://github.com/okta/okta-sdk-nodejs/blob/master/src/generated-client.js#L19) into GeneratedClient using Node syntax and [referenced](https://github.com/okta/okta-sdk-nodejs/blob/master/src/generated-client.js#L187) in methods bodies. For type definitions we import model types that are referenced by methods signatures [explicitly](https://github.com/okta/okta-sdk-nodejs/blob/op-ts-types-support/src/types/generated-client.d.ts#L14). 

Thanks for mentioning this - it looks like some of templates do not have warning banner included - going to fix that.